### DISCUSSION TITLE
4.1 Virtual Machine Import button Server Error

### DISCUSSION THREAD

**Discussion by uppsju** [netbox-community/netbox#17433] [Q&A] [answered]
I upgraded from Netbox 4.0.6 to 4.1
When going to Virtual Machines and clicking the Import button, I get the following:
`<class 'TypeError'>

Field.__init__() got an unexpected keyword argument 'query_params'

Python version: 3.11.2
NetBox version: 4.1.0
Plugins: 
  netbox_dns: 1.1.0`

**Comment by candlerb** [comment, marked as answer]
Search in discussions for `unexpected keyword argument 'query_params'` and you'll find https://github.com/netbox-community/netbox/discussions/17415

**Reply by uppsju** [reply]
thanks!
